alembic/ddl/impl.py

```diff
@@ -136,6 +139,32 @@
         self.output_buffer.write(text + "\n\n")
         self.output_buffer.flush()
 
+    def version_table_impl(
+        self, *, version_table, version_table_schema, version_table_pk, **kw
+    ) -> Table:
+        """create the Table object for the version_table.
+
+        Provided as part of impl so that third party dialects can override
+        this.
+
+        .. versionadded:: 1.13.4
+
+        """
+        vt = Table(
```

Comment:
**Federico Caselli** (CaselIT) wrote:

it should 1.14

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/5556

Reply:
Done